keyboards/treadstone32/keymaps/default/keymap.c

```diff
@@ -0,0 +1,188 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+  _BASE = 0,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  RGBRST = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  KANJI,
+};
+
+// enum tapdances{
+//   TD_CODO = 0,
+//   TD_SLRO,
+// };
+
+// Layer Mode aliases
+#define KC_MLAD  MO(_ADJUST)
+
+#define KC______ KC_TRNS
```

Comment:
QMK already defines the shortcuts `_______` and `XXXXXXX` for `KC_TRNS` and `KC_NO`, so you can use them here instead and remove these defines. In fact you can probably remove a lot of the defines below too, as there's no need to prepend them with `KC_`.